Monika After Story/game/script-topics.rpy

```diff
@@ -12777,3 +12777,24 @@ label monika_sleigh:
     show monika 5hkbfa at t11 zorder MAS_MONIKA_Z with dissolve
     m 5hubfa "An experience like that with you would be so breathtaking~"
     return
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_enjoyingspring",category=['spring'],prompt="Enjoying spring",random=mas_isSpring()))
+
+label monika_enjoyingspring:
+    m 1esa "Spring is such an amazing time of year, isn't it?"
+    m 1hua "The cold snow finally melts away, and the sunshine brings new life to nature."
+    m 1hub "When the flowers bloom, I can't help but smile!"
+    m 3hub "It's like the plants are waking up and saying, 'Hello world!' Ahaha~"
+    m 2esa "But I think the best thing about spring, would have to be the cherry blossoms."
+    m 4eub "They're very popular all around the world, but the most popular cherry blossoms would have to be the {i}'Somei Yoshino'{/i} in Japan."
+    m 3esb "Those ones in particular are mostly white, with a slight tinge of pink."
+    m 3eub "Did you know that they only bloom for one week each year?"
+    m 3lksdla "It's quite a short lifespan, but it's still beautiful regardless."
+    m 4tkc "One big downside to spring, however, would be the constant rainfall."
+    m 3esb "But rain can also be fun too!"
+    m 3hua "We can always go for walks together in the rain, we just have to bring an umbrella big enough for two!"
```

Comment:
This bit here should *probably* be happy+, especially that second last line, seeing as holding is happy+. Would probably need an alternate ending though, just to tie it up for those at normal-.